include/battle.h

```diff
@@ -200,7 +200,7 @@ struct ProtectStruct
     u32 field3 : 8;
 
     /* field_4 */ u32 physicalDmg;
-    /* field_8 */ u32 specialDmg;
+    /* soundTimer */ u32 specialDmg;
```

Comment:
Not sure why these comments are here in the first place, but accidental F&R here